src/pages/docs/creators/ray/troubleshooting.md

```diff
@@ -61,9 +121,13 @@ It produces an output like this:
   71258 ?        Sl     2:35 yagna
 ```
 
-The above shows `ray-on-golem` webserver and the `yagna` daemon are running.
+The above shows the `ray-on-golem` webserver and the `yagna` daemon are running.
+
+Note that Ray on Golem leaves the `yagna` daemon running on purpose - it stays connected to the Golem network maintaining current information about the providers so that when you start up your cluster again the nodes are found quicker. 
+
+With that in mind, we recommend killing only `ray-on-golem` and leaving `yagna` running. 
```

Comment:
I'd put these lines after "The surest way", since only then do we mention killing those services...

Reply:
Left it where it was, but elaborated more (and removed killing ;)
```
The above shows the `ray-on-golem` webserver and the `yagna` daemon are running.

Note that Ray on Golem leaves the `yagna` daemon running on purpose - it stays connected to the Golem network maintaining current information about the providers so that when you start up your cluster again the nodes are found more quickly.

With that in mind, when starting over we recommend stopping ray-on-golem and leaving `yagna` running (but to get a truly clean slate you might want to stop `yagna` too).

The surest way to stop these services is to kill them (using the PID numbers as shown in the first column):
```